packages/core/content-releases/admin/src/components/ReleaseModal.tsx

```diff
@@ -30,15 +32,22 @@ export const ReleaseModal = ({
   isLoading = false,
 }: ReleaseModalProps) => {
   const { formatMessage } = useIntl();
+  const { pathname } = useLocation();
+  const isOnUpdate = pathname !== `/plugins/${pluginId}`;
 
   return (
     <ModalLayout onClose={handleClose} labelledBy="title">
       <ModalHeader>
         <Typography id="title" fontWeight="bold" textColor="neutral800">
-          {formatMessage({
-            id: 'content-releases.modal.add-release-title',
-            defaultMessage: 'New release',
-          })}
+          {isOnUpdate
+            ? formatMessage({
+                id: 'content-releases.modal.edit-release-title',
+                defaultMessage: 'Edit release',
+              })
+            : formatMessage({
+                id: 'content-releases.modal.add-release-title',
+                defaultMessage: 'New release',
+              })}
```

Comment:
I don't really have a preference but I think the ternary could also be handled directly in the translation. I think the upside of that is managing one translation key vs 2, wdyt?

Same for the translation below